packages/outputarea/src/widget.ts

```diff
@@ -1056,4 +1071,40 @@ namespace Private {
     name: 'preferredMimetype',
     create: owner => ''
   });
+
+  /**
+   * A `Panel` that's focused by a `contextmenu` event.
+   */
+  export class OutputPanel extends Panel {
+    /**
+     * Construct a new `OutputPanel` widget.
+     */
+    constructor(options?: Panel.IOptions) {
+      super(options);
+      this.node.tabIndex = -1;
```

Comment:
If introducing a `-1` tab index please add a comment explaining why it is needed. A recent PR moved most -1 to 0 to improve accessibility (https://github.com/jupyterlab/jupyterlab/pull/10289) and it would be best to be more conscious about it now. It might be that it is genuinely needed here as -1 - that's fine, but if so let's explain it in a comment.